content/en/tracing/trace_collection/automatic_instrumentation/single-step-apm.md

```diff
@@ -469,15 +571,27 @@ To stop producing traces, remove library injectors and restart the infrastructur
 
 {{% tab "Kubernetes" %}}
 
-1. Under `apm:`, remove `instrumentation:` and all following configuration in `datadog-values.yaml`.
-2. Run the following command:
-   ```bash
-   helm upgrade datadog-agent -f datadog-values.yaml datadog/datadog
+The file you need to configure depends on if you enabled Single Step Instrumentation with Datadog Operator or Helm:
+
+{{< collapse-content title="Datadog Operator" level="h4" >}}
+
+1. Under `apm:`, remove `instrumentation:` and all following configuration in `datadog-agent.yaml` 
```

Comment:
Can we please change this section for both Operator and Helm? If users want to disable APM Instrumentation, they should set {datadog|features}.apm.instrumentation.enabled=false.

Reply:
Can do. Also, for the Helm instructions, should `kubectl apply -f /path/to/your/datadog-agent.yaml` be `kubectl apply -f /path/to/your/datadog-values.yaml` instead?